src/main/java/net/dv8tion/jda/bot/sharding/DefaultShardManagerBuilder.java

```diff
@@ -82,6 +82,19 @@
      * before calling {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder#build() build()}.
      */
     public DefaultShardManagerBuilder() {}
+    
+    /**
+     * Creates a DefaultShardManagerBuilder with the given token.
+     * <br>This is equivalent to using the constuctor
+     * {@link net.dv8tion.jda.bot.sharding#DefaultShardManagerBuilder() DefaultShardManager()}
+     * and calling {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder#setToken(String) setToken(String)}
+     * directly afterward. You can always change the token later with 
+     * {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder#setToken(String) setToken(String)}.
```

Comment:
just do `{@link #setToken(String) ...}` on both of these

Reply:
Done